src/uct/api/v2/uct_v2.h

```diff
@@ -596,6 +596,8 @@ typedef struct uct_iface_is_reachable_params {
 /**
  * @ingroup UCT_RESOURCE
  * @brief Operation parameters passed to @ref uct_ep_is_connected.
+ *        Each transport should receive only the relevant addresses for this
```

Comment:
Let's rephrase. Try to replace "relevant" with more clear definition.
Also, it's not "each transport should receive..." better be replaced with "pass ... according to the transport"